holoviews/element/comparison.py

```diff
@@ -131,7 +131,9 @@ def register(cls):
         cls.equality_type_funcs[np.ma.masked_array]  = cls.compare_arrays
 
         # Pandas dataframe comparison
-        cls.equality_type_funcs[pd.DataFrame] = cls.compare_dataframe
+        if _is_installed("pandas"):
```

Comment:
Just noting it means that if one installs pandas after importing holoviews this compare_dataframe function won't get registered.

Reply:
Yes. 

I have never been a massive fan of this file, or more specifically, its location. I would have liked it to be in a testing module.